crates/networking/discv5/src/discovery.rs

```diff
@@ -39,6 +39,17 @@ pub struct DiscoveredPeers {
     pub peers: HashMap<Enr, Option<Instant>>,
 }
 
+#[derive(Debug)]
+pub struct UpdatedEnr {
+    pub enr: Enr,
+}
+
+#[derive(Debug)]
+pub enum DiscoveryOutEvent {
+    DiscoveredPeers(DiscoveredPeers),
+    UpdatedEnr(UpdatedEnr),
+}
```

Comment:
```suggestion
#[derive(Debug)]
pub enum DiscoveryOutEvent {
    DiscoveredPeers { peers: HashMap<Enr, Option<Instant>> },
    UpdatedEnr { enr: Enr },
}
```

What do you think of inlining these? Having structs for these fields, just seems a little overkill right now

Reply:
yup good idea